airflow/providers/amazon/aws/sensors/sqs.py

```diff
@@ -152,7 +151,7 @@ def poke(self, context: Context):
         :param context: the context object
         :return: ``True`` if message is available or ``False``
         """
-        sqs_conn = self.get_hook().get_conn()
+        sqs_conn = self.hook.get_conn()
```

Comment:
```suggestion
        sqs_conn = self.hook.conn
```

Reply:
it'd break the tests, I'll take a closer look tomorrow